blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Sourcegraph 3.17: Faster and automatic precise code intelligence, product preview of code insights, and AND/OR queries for all search modes"
+author: Adam Herzog
```

Comment:
@christinaforney who should be the author? just put myself but maybe you or @poojaj-tech?

Reply:
I have less context on release 3.17 @adaaaam so likely you or @christinaforney.